ios/brave-ios/Package.swift

```diff
@@ -292,7 +292,15 @@ var package = Package(
         .copy("LottieAssets/onboarding-rewards.json"),
         .copy("LottieAssets/onboarding-shields.json"),
         .copy("LottieAssets/playlist-confetti.json"),
-        .copy("Welcome/Resources/disconnect-entitylist.json"),
+        .copy("WelcomeFocus/Resources/LottieAssets/moving-ads-dark.json"),
```

Comment:
We dont need these I guess. Why do we add them? They are used in new focus onboarding which is for 1.67.